pants.toml

```diff
@@ -15,10 +15,17 @@ backend_packages.add = [
   'pants.backend.python.lint.isort',
 ]
 
+# This ensures that the result of `./pants binary` is always unique.
+pants_distdir_legacy_paths = false
+
 [source]
 # The Python source root is the repo root. See https://www.pantsbuild.org/docs/source-roots.
 root_patterns = ["/"]
 
+[subprocess-environment]
+# We pass through these environment variables to subprocesses.
+env_vars = ["LANG", "LC_ALL"]
```

Comment:
Both of the edits in `pants.toml` seem like maybe they would make good defaults... especially this one, since it seems like Python code will break without them?

Reply:
Strong +1 on this one with `LANG` and `LC_ALL`. Originally, I was concerned that it's then hard to add your own values to this option. But I realized that I'm not concerned about that, thanks to the syntax `env_vars.add`.

@benjyw what do you think?